src/analyses/goto_check.h

```diff
@@ -58,7 +58,7 @@ void goto_check(
   " --no-built-in-assertions     ignore assertions in built-in library\n" \
   " --enum-range-check           checks that all enum type expressions have values in the enum range\n" /* NOLINT(whitespace/line_length) */ \
   " --pointer-primitive-check    checks that all pointers in pointer primitives are valid or null\n" /* NOLINT(whitespace/line_length) */ \
-  " --allocate-size-check        checks that not more memory is allocated than can be addressed by cbmc" /* NOLINT(whitespace/line_length) */
+  << help_entry("--allocate-size-check", "checks that not more memory is allocated than can be addressed by cbmc") /* NOLINT(whitespace/line_length) */
```

Comment:
What is the `help_entry` function. Where was it introduced? Is there a CBMC equivalent?

Reply:
It's defined in `parse_options.cpp`. I've introduced it a few weeks ago. It's supposed to make writing the help entries easier, by automatically wrapping the description at 80 characters, and indenting it by 30 characters.